Makefile

```diff
@@ -109,7 +109,7 @@ dag:
 	  --profile=profiles/$(PROFILE) \
```

Comment:
I had to update it in the end because the quantization broke. This might have something to do with the new vocab settings, not sure. I don't think we want to review the diff for the Marian update. We'll need to test the compatibility of the new models with our current WASM module just in case, but I don't think it can be broken. It's one line, so I don't know what would be the purpose of adding another PR for this.